### ISSUE TITLE
Should UnitConv objects include strings for the unit names?

### ISSUE THREAD

**Issue by willrogers** [dls-controls/pytac#89]
e.g. BPMs convert between `mm` and `m`; magnets typically convert between `A` and some magnetic field.

**Comment by T-Nicholls** [comment]
fields: PHYS <-> ENG
x&y: mm <-> m
f: Hz <-> Hz
x&y_kick: 100th of a radian? <-> degrees?
b0&b1&b2&a1: field strength(Tesla, Gauss?) <-> A
These are all our element fields and my guesses at their respecive units.

**Comment by T-Nicholls** [comment]
fields: PHYS <-> ENG (according to the internet)
m44: ? <-> same
s_position: m <-> m
alpha: ? <-> same
beta: ? <-> same (Beta has units of length?)
mu: ? <-> same
dispersion: ? <-> same (For instance in LEP, where the dispersion is of the order of a metre?)
tune_x&y: ? <-> same (The “tune” or Q value, often denoted also with v)
chromaticity_x&y ? <-> same (Figure 3.9a shows the actual chromaticity measurements converted to units of b2 as a function of energy?)
energy: Mev <-> Mev
emittance_x&y: ? <-> same (Emittance has units of length, but is usually referred to as "length × angle", for example, "millimeter × milli-radians".)
beam_current: I? <-> same
This is my attempt at the lattice fields, both live and sim.

**Comment by T-Nicholls** [comment]
In the process of being implemented through pull request #98.

**Comment by willrogers** [comment]
You can find at least some of these in Middlelayer:

```
>> ao = getao;
>> ao.Q1B.Monitor

ans = 

                Mode: 'Online'
               Units: 'Physics'
             HWUnits: 'a'
            DataType: 'scalar'
        ChannelNames: [34x17 char]
               Range: [34x2 double]
        PhysicsUnits: 'm^-2'
       HW2PhysicsFcn: @amp2k
       Physics2HWFcn: @k2amp
    HW2PhysicsParams: [34x1 double]
    Physics2HWParams: [34x1 double]
```

**Comment by T-Nicholls** [comment]
Still unknown:
- m44
- alpha
- mu
- dispersion
- tune
- chromaticity
- a1, b1 etc... mml has them as m^-something but should field strength be a component?

**Comment by willrogers** [comment]
Units of angles are dimensionless.

My current best guesses:

* m44 - dimensionless
* mu - dimensionless
* dispersion - m
* tune - dimensionless
* chromaticity - dimensionless
* alpha - dimensionless
* beta - m
* gamma m^-1

Because of the way that ax and bx are normalised, the MML units are correct.

**Comment by T-Nicholls** [comment]
Now fully implemented in #98.
